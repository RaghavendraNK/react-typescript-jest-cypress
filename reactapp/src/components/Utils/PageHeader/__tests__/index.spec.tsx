import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { PageHeader, Props } from '../index';

describe('PageHeader', () => {
    let wrapper: ShallowWrapper;
    let props: Props;
    beforeAll(() => {
        props = {
            header: 'header',
            description: 'description',
        };
        wrapper = shallow(<PageHeader {...props} />);
    });
    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
