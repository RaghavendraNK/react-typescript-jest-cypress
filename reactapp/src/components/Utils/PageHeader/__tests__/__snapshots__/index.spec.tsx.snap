// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader should match snapshot 1`] = `
<div
  id="pageHeader"
>
  <div
    className="page-header"
  >
    <C
      size={4}
      text="header"
    />
  </div>
  <div
    className="page-header-helper"
  >
    <g
      size={2}
      text="description"
    />
  </div>
</div>
`;
